/**:
  ros__parameters:
    origin_lat: 35.326441528
    origin_lon: 139.34767431
    lanelet2_file_full_path: /home/ataparlar/data/openstreetmap_data/hiratsuka_small/commonroad_data/lanelet2_map_hiratsuka_200m_4326.osm
    output_lanelet2_file_full_path: /home/ataparlar/data/openstreetmap_data/hiratsuka_small/commonroad_data/lanelet2_map_hiratsuka_200m_4326_test1.osm